src/features/channels/components/message-input/MessageInput.tsx

```diff
@@ -29,6 +28,10 @@ const MessageInput = () => {
   }, [selectedChannel]);
 
   const onSubmit = ({ message }: FormData) => {
+    if (!message.trim()) {
```

Comment:
same as above comment, just make sure that 'message' is not nullable, which means that it will always contain a value, or empty string

Reply:
Yes, due to linked with an input which value only can be a string